apps/cookbook/src/app/showcase/alert-showcase/alert-showcase.component.html

```diff
@@ -37,7 +37,30 @@
     ...
 {{ '}' }}</cookbook-code-viewer>
   </p>
-  <h4>Alert config properties:</h4>
+
+  <h2>Accessibility</h2>
+  <p>
+    Alerts have a
+    <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role">
```

Comment:
```suggestion
    <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role">dialog</a>
```

Reply:
I understand now! This is an issue we have because our prettier-config adds newlines for html tags, at least when above a certain amount of characters.

Added prettier-ignore and put it on one line as you suggested.